src/test/java/org/opentripplanner/routing/trippattern/DeduplicatorTest.java

```diff
@@ -187,14 +204,7 @@ public void testToString() {
     subject.deduplicateImmutableList(DATE_CL, DATE_LIST);
 
     assertEquals(
-        "Deduplicator{"
-            + "BitSet: 1(1), "
-            + "IntArray: 1(1), "
-            + "String: 2(2), "
-            + "StringArray: 1(1), "
-            + "LocalDate: 1(2), "
-            + "List<LocalDate>: 1(1)"
-            + "}",
+        "Deduplicator{BitSet: 1(1), IntArray: 1(1), String: 2(2), StringArray: 1(1), String2DArray: 0(0), LocalDate: 1(2), List<LocalDate>: 1(1)}",
```

Comment:
Keep the previous formatting, the line is too long.

Reply:
This is now changed.